caffeine@patapon.info/extension.js

```diff
@@ -545,36 +554,31 @@ class Caffeine extends QuickSettings.SystemIndicator {
     }
 
     _startTimer() {
-        if(this._settings.get_boolean(TIMER_ENABLED_KEY)) {
-            this._timerEnable = true;
-
-            // Reset Timer
-            this._removeTimer(true);
+        this._timerEnable = true;
 
-            // Enable Caffeine
-            this._settings.set_boolean(TOGGLE_STATE_KEY, true);
+        // Reset timer
+        this._removeTimer(true);
```

Comment:
I believe if you call first `_removeTimer()` before `this._timerEnable = true;` you don't need to handle a `reset` param in `_removeTimer()`

Reply:
Yes, you're right. I'll rewrite this. :-)